.github/workflows/gorgone.yml

```diff
@@ -285,7 +289,7 @@ jobs:
     runs-on: [self-hosted, common]
     strategy:
       matrix:
-        distrib: [el8, el9, bullseye, bookworm]
+        distrib: [el8, el9, bullseye, bookworm, jammy]
```

Comment:
```suggestion
        distrib: [el8, el9, bullseye, bookworm]
```

Reply:
Please avoid editing promote job in general since this will publish jammy version to stable